plugins/flytekit-omegaconf/flytekitplugins/omegaconf/dictconfig_transformer.py

```diff
@@ -0,0 +1,194 @@
+import importlib
+import logging
+import re
+import traceback
+import typing
+from typing import Type, TypeVar
+
+import flatten_dict
+from flyteidl.core.literals_pb2 import Literal as PB_Literal
+from flytekitplugins.omegaconf.config import OmegaConfTransformerMode, SharedConfig
+from flytekitplugins.omegaconf.flytekit_patch import iterate_get_transformers
+from flytekitplugins.omegaconf.type_information import extract_node_type
+from google.protobuf.json_format import MessageToDict, ParseDict
+from google.protobuf.struct_pb2 import Struct
+
+import omegaconf
+from flytekit import FlyteContext
+from flytekit.core.type_engine import TypeTransformerFailedError
+from flytekit.extend import TypeEngine, TypeTransformer
+from flytekit.models.literals import Literal, Scalar
+from flytekit.models.types import LiteralType, SimpleType
+from omegaconf import DictConfig, OmegaConf
+
+logger = logging.getLogger("flytekit")
+
+T = TypeVar("T")
+NoneType = type(None)
+
+
+def is_flatable(d: DictConfig) -> bool:
```

Comment:
Nit (non-blocking):
Should maybe called `is_flattenable`.

Reply:
agreed, addressed in https://github.com/flyteorg/flytekit/pull/2299/commits/08f9a8092adc6943e13c474250c72ceb36f96a45